packages/discovery/Discovery.yml

```
@@ -279,7 +279,7 @@ apis:
         name: Assisted-Install Service
         description: Assisted installation
         url: https://api.openshift.com/api/assisted-install/v2/openapi
-        apiType: openapi-v2
+        apiType: openapi-v3
```

```suggestion
        apiType: openapi-v3
        useLocalFile: true
```

We also need to tell the process to use the local file

---

PR updated.  Sorry, I missed the option to just commit your suggestion.  I'll get that next time.